src/cdk/scrolling/viewport-ruler.ts

```diff
@@ -20,9 +28,22 @@ export class ViewportRuler {
   /** Cached document client rectangle. */
   private _documentRect?: ClientRect;
 
-  constructor(scrollDispatcher: ScrollDispatcher) {
+  /** Stream of viewport change events. */
+  private _changed = new Subject<string>();
```

Comment:
Is the difference between `resize` and `orientationchange` useful here? If we make it `void` we can change it to a more robust object later without it being a breaking change

Reply:
The intention was to let people react differently depending on the event, but we can always change it if somebody asks for it.